.github/workflows/test.yaml

```
@@ -22,97 +14,32 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 20
     steps:
-      - name: Remove preview consumption comment
-        if: github.event_name == 'pull_request'
-        uses: marocchino/sticky-pull-request-comment@773744901bac0e8cbb5a0dc842800d45e9b2b405 # v2.9.4
-        with:
-          header: pr_preview_consumption
-          delete: true
-          GITHUB_TOKEN: ${{ secrets.SPEC_RENDERER_BOT_PAT }}
-
       - name: Checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
-        with:
-          fetch-depth: 0
 
       - name: Setup PNPM with Dependencies
         uses: ./.github/actions/setup-pnpm-with-dependencies/
-        with:
-          force-install: true
 
       - name: Stylelint
-        run: pnpm run stylelint
+        run: |
+          echo "${{github.event_name}}"
+          pnpm run stylelint
 
       - name: Lint
         run: pnpm run lint
 
       - name: Build
         run: pnpm run build
 
+      - name: Test
+        run: pnpm run test
+
       - name: Upload build artifacts
-        if: ${{ github.event_name == 'push' }}
+        if: |
+          github.event_name != 'workflow_dispatch'
```

Any reason not to use the normal syntax?
```suggestion
        if: ${{ github.event_name != 'workflow_dispatch' }}
```